integration/live_update_after_crash_rebuild_test.go

```diff
@@ -0,0 +1,76 @@
+//+build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLiveUpdateAfterCrashRebuild(t *testing.T) {
+	f := newK8sFixture(t, "live_update_after_crash_rebuild")
+	defer f.TearDown()
+	f.SetRestrictedCredentials()
+
+	f.TiltWatch()
+
+	ctx, cancel := context.WithTimeout(f.ctx, time.Minute)
+	defer cancel()
+	fmt.Println("> Waiting for pods from initial build")
+	oneUpPods := f.WaitForAllPodsReady(ctx, "app=live-update-after-crash-rebuild")
+
+	ctx, cancel = context.WithTimeout(f.ctx, time.Minute)
+	defer cancel()
+	f.CurlUntil(ctx, "http://localhost:31234", "🍄 One-Up! 🍄")
+
+	// Live update
+	f.ReplaceContents("compile.sh", "One-Up", "Two-Up")
+
+	ctx, cancel = context.WithTimeout(f.ctx, time.Minute)
+	defer cancel()
+	fmt.Println("> Perform a live update")
```

Comment:
nit: I feel like this should be above the `ReplaceContents`, so that it's logged before the live update starts, rather than potentially after it's already done